itest/loadtest/loadtest-sample.conf

```diff
@@ -22,6 +22,10 @@ send-test-num-sends=5
 # Number of assets to send in each send operation for send test
 send-test-num-assets=1
 
+# Type of asset to use in the test case. For V2 test cases, a 0 type is
+# required.
+send-test-send-type=0
+
```

Comment:
Is 0 here the `asset.Type` `Normal`? I think this should be a string rather than an int. Maybe something like:
```
send-test-asset-type=normal
```